src/borg/helpers/parseformat.py

```diff
@@ -457,6 +457,11 @@ def validator(text):
             raise argparse.ArgumentTypeError('"%s": No archive specified' % text)
         elif archive is False and loc.archive:
             raise argparse.ArgumentTypeError('"%s" No archive can be specified' % text)
+        if proto is not None and loc.proto != proto:
+            if proto == 'file':
+                raise argparse.ArgumentTypeError('"%s" must be a local repository' % (text))
+            else:
+                raise argparse.ArgumentTypeError('"%s" must be a remote repository' % (text))
```

Comment:
maybe keep same style as in 457 and 459, to be consistent.

(foo) is pointless anyway as it is not a 1-tuple, for that it would need (foo, ).